.travis.yml

```
@@ -1,7 +1,7 @@
 language: java
 sudo: false
 jdk:
-- oraclejdk8
+- oraclejdk10
```

How about a matrix build:
```suggestion
- oraclejdk8
- oraclejdk9
- oraclejdk10
```